library/systemd/src/lib/yast2/compound_service.rb

```diff
@@ -0,0 +1,202 @@
+require "yast"
+
+require "yast2/system_service"
+
+module Yast2
+  # Class that represents several related services that need to be synchronized.
+  # It mimics behavior of single SystemService, just adds additional possible states.
+  class CompoungService
+    include Yast::Logger
+
+    # managed services
+    # @return [Yast2::ServiceConfiguration]
+    attr_reader :services
+
+    # creates new configuration that holds state for given services
+    # @param services<Yast::SystemdServiceClass::Service> services to configure
+    # @param reload<true,false> if use reload instead of restart action. If
+    #   service does not support reload or does not run, then restart is used.
+    #
+    # @example three services
+    #   config = Yast2::ServiceConfiguration.new(s1, s2, s3)
+    def initialize(*services)
+      if services.any? { |s| !s.is_a?(Yast2::SystemdService) }
+        raise ArgumentError, "Services can be only System Service - #{services.inspect}"
+      end
+
+      @services = services
+    end
+
+    # writes services new status
+    # @raise <Yast::SystemctlError> when set service to target state failed
+    def save(ignore_status: false)
```

Comment:
NP: In SystemService I have rename this param as `keep_state`. Maybe we should do the same here.

Reply:
yes, plan is to have identical API.